apps/meteor/imports/personal-access-tokens/server/api/methods/regenerateToken.ts

```diff
@@ -39,6 +39,8 @@ Meteor.methods<ServerMethods>({
 		await Meteor.callAsync('personalAccessTokens:removeToken', { tokenName });
 		return Meteor.callAsync('personalAccessTokens:generateToken', {
 			tokenName,
+			// @ts-expect-error - This doesn't make sense. We're querying a user and expecting Mongo to return a token. We should look into this one
```

Comment:
The wrong code was already here before, and then the [type was changed](https://github.com/RocketChat/Rocket.Chat/pull/28789/files#diff-b6b74677850bdf2222dac607cf3887c922033bfc354f83ed54437b4bf39581a2R241) to match the wrong code. You can switch it back to `IUser` and leave a comment there.

Reply:
Nice, i typed it as IUser for now. I'll keep track of this one to fix it on another PR

Thx man!